internal/app/export_test.go

```diff
@@ -14,11 +14,13 @@ type MockAuthorizer struct {
 
 // MockProxy is a mock proxy.
 type MockProxy struct {
-	ApplyError bool
+	ApplyCount   int
+	ApplyError   bool
+	SleepOnApply time.Duration
 }
 
 // IsSenderAllowed is a mock implementation of authorizerer, returning an error if requested in the mock.
-func (m *MockAuthorizer) IsSenderAllowed(ctx context.Context, action string, sender dbus.Sender) (err error) {
+func (m *MockAuthorizer) IsSenderAllowed(action string, sender dbus.Sender) (err error) {
```

Comment:
Usually, when I see this kind of function name, I expect a answer to a question (i.e. `true | false`) instead of an error. Maybe we can thing of a better name for it? Something in the likes of `CheckSender`? (I suck at naming, but I'm sure we can find a good name with three brains working together)

Reply:
This is true.

`CheckSenderAllowed` maybe (as we don’t check for the sender validity, but the fact that the sender can perform that action). We need to update adsys too then!